docs/landscape.html

```diff
@@ -876,6 +877,9 @@ <h3><a name="Landscape">Landscape</a></h3>
      <li>m2: index into the array of stations</li>
      <li>m3 bits 7..4: persistent random data for railway stations/waypoints and airports)</li>
      <li>m3 bits 7..4: <a href="#OwnershipInfo">owner</a> of tram tracks (road stop)</li>
+     <li>m3 bit 2: rail station / waypoint may have catenary pylons</li>
+     <li>m3 bit 1: rail station / waypoint may have catenary wires</li>
+     <li>m3 bit 0: rail station / waypoint is blocked</li>
      <li>m3 bits 3..2: ground type (road waypoints)
```

Comment:
```suggestion
     <li>m3 bits 3..2: ground type (road waypoints)
     <li>m3 bit 2: rail station / waypoint may have catenary pylons</li>
     <li>m3 bit 1: rail station / waypoint may have catenary wires</li>
     <li>m3 bit 0: rail station / waypoint is blocked</li>
```
maybe for better 7 to 0 ordering.